docs/UserGuide.md

```diff
@@ -195,7 +196,7 @@ Sets a person's status as favourite or blacklisted.
 
 Format: `status INDEX s/STATUS`
 - Gives a status to the person at specified `INDEX`
-- Status can either be a `blacklist` or `favourite`, a person can have no status tagged.
+- Status can either be a `blacklist` or `favourite`, a person can have no status module.
```

Comment:
Unintentional side effect of mass replacement?

Reply:
Good catch, it was a mistake.